### DISCUSSION TITLE
Transporter & Expression duplication by cloning and loading as a json file

### DISCUSSION THREAD

**Discussion by GhazalMontaseri** [Open-Systems-Pharmacology/Forum#1818] [Questions & Problems]
Dear OSPS community, I have used Pk-Sim V11.2. Consider a compound with Pgp as transporter. When I clone a simulation, save and then load the model as a json file, in the cloned simulation (configure), pgp is duplicated (one with None in front  which means inactive and one which is active.). If I now clone a cloned simulation and again save as json and load it, this time in the 'process' tab of the configuration, I see triple pgp (as seen below). mathematically this does not do sth wrong as the duplicates are set to None but they are redundant. Any idea how to solve this issue?
![pk-sim_dubplicate](https://github.com/user-attachments/assets/ed9e68b5-c281-45aa-b71e-9e86af7df6bf)

**Comment by GhazalMontaseri** [comment]
Dear @Yuri05 any help in this question?

**Reply by Yuri05** [reply]
Hi @GhazalMontaseri , I cannot reproduce the issue in V11.3 and assume it was fixed.
I would suggest upgrading to V11.3 (or wait for 1-2 weeks till V12 is released :))
